drivers/mrf24j40/mrf24j40_getset.c

```diff
@@ -329,6 +329,54 @@ void mrf24j40_set_cca_threshold(mrf24j40_t *dev, int8_t value)
     mrf24j40_reg_write_short(dev, MRF24J40_REG_CCAEDTH, RSSI_value[value]);
 }
 
+int8_t mrf24j40_get_ed_level(mrf24j40_t *dev)
```

Comment:
This function basically re-implements `mrf24j40_cca()`.

Just change `bool mrf24j40_cca(mrf24j40_t *dev)` to `bool mrf24j40_cca(mrf24j40_t *dev, uint8_t *rssi, uint8_t *ccaedth)` and you'll end up with a much smaller patch and less duplicated code.

Reply:
I pushed the changes and also removed the bitfield definitions of GPIO register as it's in PR #11410.